### ISSUE TITLE
Operator adds opentsdb-http port to endpoints in VMServiceScrape

### ISSUE THREAD

**Issue by skibadmitriy** [VictoriaMetrics/operator#420]
Hello

I've run into a problem with unexpected behavior of VMOperator when I've enabled insertPorts.openTSDBHTTPPort: "4242 in VMCluster spec. VMOperator has added this port (opentsdb-http) to VMServiceScrape of VMInsert
```
spec:
  endpoints:
  - path: /metrics
    port: http
  - path: /metrics
    port: opentsdb-http
```
But this port isn't suitable for getting metrics and doesn't have /metrics path. So I have a lot of error messages in VMAgent log

**Comment by f41gh7** [comment]
Thanks for reporting, will be fixed soon.

**Comment by skibadmitriy** [comment]
I have operator version 
`
dmitriy@dmitriy-20rd001drt:~/app/edna/cd/cluster-system-apps$ kubectl -n vm-operator exec -it vm-operator-59bc5475b7-8tz4v -- sh
~ $ manager 
bin/    dev/    etc/    home/   lib/    media/  mnt/    opt/    proc/   root/   run/    sbin/   srv/    sys/    tmp/    usr/    var/
~ $ manager --version
v0.22.0

`

**Comment by skibadmitriy** [comment]
VMserviceScrape was created by operator
```
dmitriy@dmitriy-20rd001drt:~/app/edna/cd/cluster-system-apps$ kubectl -n vm-operator get vmservicescrapes.operator.victoriametrics.com vminsert-vmcluster -o yaml
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMServiceScrape
metadata:
  creationTimestamp: "2022-01-28T09:20:36Z"
  generation: 2
  labels:
    app.kubernetes.io/component: monitoring
    app.kubernetes.io/instance: vmcluster
    app.kubernetes.io/name: vminsert
    managed-by: vm-operator
    vmagent: vmagent
  name: vminsert-vmcluster
  namespace: vm-operator
  ownerReferences:
  - apiVersion: operator.victoriametrics.com/v1beta1
    blockOwnerDeletion: true
    controller: true
    kind: VMCluster
    name: vmcluster
    uid: b8dc7c17-6fde-4d27-89c0-c91bfc472935
  resourceVersion: "10232749"
  uid: 19bc05ad-c410-4554-8d00-8dabd56ff63a
spec:
  endpoints:
  - path: /metrics
    port: http
  - path: /metrics
    port: opentsdb-http
  namespaceSelector: {}
  selector:
    matchLabels:
      app.kubernetes.io/component: monitoring
      app.kubernetes.io/instance: vmcluster
      app.kubernetes.io/name: vminsert
      managed-by: vm-operator
```

**Comment by f41gh7** [comment]
Changes was included to the v0.23.3 release. Feel free to re-open the issue if it doesn't solve it.
